docs/QUICKSTART.md

```diff
@@ -68,8 +68,8 @@ void connectInsecure(String host, int port) {
 ```kotlin
 fun fetch() {
     lifecycleScope.launch {
-        val property = Property("Vehicle.Speed", listOf(Field.FIELD_VALUE))
-        val response = dataBrokerConnection?.fetch(property) ?: return@launch
+        val request = FetchRequest("Vehicle.Speed", Field.FIELD_VALUE)
```

Comment:
Could we call this GetRequest as in the gRPC kuksa.val.v1 interface exposed by the databroker? (https://github.com/eclipse/kuksa.val/blob/master/proto/kuksa/val/v1/val.proto).

In the same way, I would prefer get() instead of fetch()

Reply:
Hey @eriksven! No strong opinion here. This was also something I thought about but decided against. We definitely want to align with the VSS terminology but not necessarily with the Kuksa gRPC one because the definition there may not fit with every other environment (Android in this case) like you already mentioned in your comment. :)

It basically just comes down to the consistent semantic we decided in the Android API here.
1) GET (and SET)  implies a really simple logic -> no overhead when retrieving or setting the information 
2) FETCH implies a more complex process like some other object will take care of the Request for you and asynchronously returns the data at some point.

That the "Request" + "Subscribe" naming fits with the gRPC one is basically just a coincidence.^^

I hope I made the reasoning a bit understandable.